src/NuGet.Clients/NuGet.CommandLine/Program.cs

```diff
@@ -149,15 +154,100 @@ public static int MainCore(string workingDirectory, string[] args)
             return 0;
         }
 
-        private static void SetConsoleOutputEncoding(System.Text.Encoding encoding)
+        /// <summary>
+        /// Takes command line args and parses response files prefixed with the @ symbol.  
+        /// Response files are text files that contain command line switches. 
+        /// Each switch can be on a separate line or all switches can be on one line.  
+        /// Multiple response files can be defined in the args. 
+        /// Nested response files are supported, but limited to a depth of 3.  
+        /// A response file cannot be larger than 2mb. 
+        /// </summary>
+        /// <param name="args">The args to parse which can contain response files</param>
+        /// <returns></returns>
+        public static string[] ParseArgsResponseFiles(string[] args)
```

Comment:
I would expect the response file parsing to be in new helper utility class instead of program.cs

Reply:
I will move to a new class